scope.go

```diff
@@ -103,6 +129,21 @@ func (s *standardScope) Report(r StatsReporter) {
 	// we do nothing for timers here because timers report directly to ths StatsReporter without buffering
 }
 
+// reportLoop is used by the root scope for periodic reporting
+func (s *standardScope) reportLoop(interval time.Duration) {
+	ticker := time.NewTicker(interval)
+	for {
+		select {
+		case <-ticker.C:
+			s.registry.sm.Lock()
```

Comment:
So we are either writing with registry.add, or we are reading from reportLoop. reportLoop is only called from one goroutine, so there's no point in using an RWMutex.